packages/client/mutations/AddFeatureFlagMutation.ts

```diff
@@ -0,0 +1,37 @@
+import {commitMutation} from 'react-relay'
+import graphql from 'babel-plugin-relay/macro'
+import {StandardMutation} from '../types/relayMutations'
+import {AddFeatureFlagMutation as TAddFeatureFlagMutation} from '../__generated__/AddFeatureFlagMutation.graphql'
+
+graphql`
+  fragment AddFeatureFlagMutation_notification on AddFeatureFlagPayload {
+    user {
```

Comment:
It looks like we do need to include the id so that Relay can find the object and update the values ([see here](https://relay.dev/docs/guided-tour/updating-data/graphql-mutations/#updating-data-once-a-request-is-complete)). 

However, [here](https://github.com/ParabolInc/parabol/blob/cf55ed4d0877b1756ffc7213062dde5056ed662b/packages/client/mutations/UpdatePokerScopeMutation.ts#L53) we don't query `id` within `user`, although perhaps that's OK as it's nested within `scores` which contains `userId`.

Reply:
That's what I thought, thanks for checking.